src/main/java/core/basesyntax/Square.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax;
+
+public class Square extends GetFigure {
+    private static final String NAME = "SQUARE";
```

Comment:
delete name from all figures, it doesn't add anything useful

Reply:
Why? I disagree with it...

Reply:
Code should contain only what is required to solve the task. Random constant that is not used doesn't help to solve the task. In same way we may add some emojis or links to favourite memes